client/app/bundles/course/learning-map/components/ZoomActionElements/index.jsx

```diff
@@ -25,19 +26,19 @@ const ZoomActionElements = (props) => {
   return (
     <div style={styles.wrapper}>
       <Button
-        label="Zoom In"
-        color="primary"
         className="btn-submit"
+        color="primary"
+        label="Zoom In"
```

Comment:
Extract to `translations.intl.js`?

Reply:
`yarn lint-fix` rearranges all the props in this PR. The initial purpose of this PR is to fix the linting, and not extract the intl. I suspect there's other labels that are not intl-ed yet. I will fix the ones highlighted in the comment but maybe another new issue should be opened to properly go through all the code and extract the labels.